src/UserBundle/DataGrid/ApiTokenGrid.php

```
@@ -0,0 +1,131 @@
+                    if ($token instanceof ApiToken) {
+                        $tokenUser = $token->getUser();
+                        assert($tokenUser instanceof User);
+                        if ($tokenUser->getId() === $currentUser->getId()) {
+                            $repository->revoke($token);
+                        }
+                    }
+                }
+            });
+    }
+
+    public function query(EntityManagerInterface $entityManager, Query $query): Query
```

PHPMD (2.15.0)

Avoid unused parameters such as '$entityManager'. (undefined)

(UnusedFormalParameter)